jwql/website/apps/jwql/views.py

```diff
@@ -49,17 +49,47 @@
 from .data_containers import get_current_flagged_anomalies
 from .data_containers import get_proposal_info
 from .data_containers import random_404_page
+from .data_containers import get_table_view_components
 from .data_containers import thumbnails_ajax
 from .data_containers import data_trending
 from .data_containers import nirspec_trending
 from .forms import AnomalySubmitForm, FileSearchForm
 from .oauth import auth_info, auth_required
+from jwql.database.database_interface import load_connection
 from jwql.utils.constants import JWST_INSTRUMENT_NAMES, MONITORS, JWST_INSTRUMENT_NAMES_MIXEDCASE
 from jwql.utils.utils import get_base_url, get_config
 
 FILESYSTEM_DIR = os.path.join(get_config()['jwql_dir'], 'filesystem')
 
 
+def db_table_viewer(request):
+    """Generate the JWQL Table Viewer view.
```

Comment:
Could you add some blank lines between the docstring sections?  Else I think `numpydoc` might throw up on this.